components/gauges/arc/overview.md

```diff
@@ -46,33 +36,47 @@ The <a href = "https://www.telerik.com/blazor-ui/arc-gauge" target="_blank">Tele
 </TelerikArcGauge>
 ````
 
-## Features
+## Scale
 
-The Telerik Arc Gauge for Blazor exposes the following features:
+The scale of the arc gauge renders the values of the [pointers]({%slug arc-gauge-pointers%}) and [labels]({%slug arc-gauge-labels%}). See the [Scale]({%slug arc-gauge-scale%}) article for more information on how to customize the scale of the component.
```

Comment:
Prefer capital letters when talking about the component as a product. Also consider mentioning some preliminary information, for example - the scale allows the developer to configure min/max values, angles and ticks. This will increase the value of the Overview article and you can even use more relevant keywords for the article link. See the article links at https://docs.telerik.com/blazor-ui/components/scheduler/overview

```suggestion
The scale of the Arc Gauge renders the values of the [pointers]({%slug arc-gauge-pointers%}) and [labels]({%slug arc-gauge-labels%}). See the [Scale]({%slug arc-gauge-scale%}) article for more information on how to customize the scale of the component.
```